app/views/bin/index.html.slim

```diff
@@ -14,63 +14,76 @@
 - if @chord_sheets.present?
   h2.title.is-3 Deleted Chord Sheets
 
-  .is-flex.is-flex-wrap-wrap.is-justify-content-center style="gap: 2rem;"
-    - @chord_sheets.each do |chord_sheet|
-      div style="border: 1px solid lightgrey; border-radius: 1rem; padding: 1rem; min-width: 40vw"
-        .is-flex.is-justify-content-space-between.is-flex-wrap-wrap style="gap: .5rem"
-          p
-            b Name:&nbsp;
-            span = chord_sheet.name
+  .mb-6 data-controller="search show-more" style="gap: 2rem;"
```

Comment:
Is the `show-more` controller required here? I can't see that it's doing anything, let's remove it if that's the case 🙂